lte/gateway/c/core/oai/common/state_converter.cpp

```diff
@@ -234,5 +234,31 @@ void StateConverter::proto_to_hashtable_uint64_ts(
   }
 }
 
+// Converts Map<uint64_t,uint64_t> to Proto (replaces hashtable to proto)
+void StateConverter::map_uint64_uint64_to_proto(
```

Comment:
just as a note, these will be not necessary after we use the protobuf generated maps instead, but let's keep this for now for the other usages